src/jdk.jlink/share/classes/jdk/tools/jlink/internal/plugins/OrderResourcesPlugin.java

```diff
@@ -46,23 +46,19 @@
  *
  * Order Resources plugin
  */
-public final class OrderResourcesPlugin implements Plugin {
+public final class OrderResourcesPlugin extends DocumentedPlugin {
     public static final String NAME = "order-resources";
```

Comment:
This is not related to your change.   It can be private static final field.   In my past observation, this static final NAME field is defined in every plugin which isn't really a need for it (at least in the current implementation).   It might be good to consider the clean up to remove this static final field and instead call `getName()` to get the name.  Just a thought.